### PR TITLE
Add auto-generated product layout

### LINKED ISSUE #111: Add auto-rendered product layout
Add auto-rendered product layout where the product data is rendered based on [product drop](https://www.voog.com/developers/markup/ecommerce-objects/product) data.

It should look like current "Product" layout. Use new [image](https://www.voog.com/developers/markup/tags/image) or [image_data](https://www.voog.com/developers/markup/tags/image-data) tag for product cover image.

It content type must be `product`:

```json
{
  "content_type": "product",
  "component": false,
  "file": "layouts/auto_generated_product.tpl",
  "layout_name": "auto_generated_product",
  "title": "Auto-generated product"
}
```

See also buy button rendering example in [Nuuk design](https://github.com/Voog/design-nuuk/blob/master/components/buy-button.tpl).

## TODO

* [ ] Add "Auto-generated product" layout
* [ ] Check and improve "[product list](https://www.voog.com/support/online-store/product-list-content-area)" widget styles when needed.

### PR DESCRIPTION
**Pull request by pxska**
Adds an auto-generated product layout to the design based on the initial product layout. Also add a browser hack to render crispy images in Chrome.

Closes #111.

### CODE REVIEW

**Review by tanelj — APPROVED** [review]
Nice work!

File: layouts/auto_generated_product.tpl [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+
+  <div class="holder">
+    {% include "header" %}
+
+    <main class="main" data-search-indexing-allowed="true">
+      <div class="main-inner">
+        <div class="content-wrap">
+          {% if page.level != 0 %}
+            {% include "submenu-for-current" exclude_products: true %}
+          {% endif %}
```
Comment (PeeterPaal) [inline layouts/auto_generated_product.tpl:36]: `submenu-for-current` is not needed as subpages cannot be added to auto-rendered product pages

**Review by PeeterPaal — CHANGES_REQUESTED** [review]
Overall looks good, but a few comments

**Review by PeeterPaal — APPROVED** [review]
Looking good!

**Review by tanelj — APPROVED** [review]
Nice work!

**Review by PeeterPaal — APPROVED** [review]
Looks nice!

**Pull request merged**